students/ChrisH/lesson05/simple.py

```diff
@@ -0,0 +1,49 @@
+# simple.py
+
+import logging
+import logging.handlers
+
+import datetime
+
+
+format = "%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s"
```

Comment:
`format` is a built in function.  You'll want to use a different var name.

Reply:
Oops. I do want to use a different name. Well, now I do.